pixi.js/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for Pixi.js v4.3.0
-// Project: https://github.com/pixijs/pixi.js/
-// Definitions by: Clark Stevenson <https://github.com/clark-stevenson>
+// Type definitions for Pixi.js 4.3.4
```

Comment:
Yeah, no problem. People can use special compatibility plugin that adds new API to older versions of lib, so this definition will work even for any 4.x